File: PRAK404-2310817310008-MUHAMMADRIZKIRAMADHAN.c
#include <stdio.h>

int main(){
    float a, b, c;
    while (1){
    printf("Pilih program\n");
    printf("1. Penjumlahan\n");
    printf("2. Pengurangan\n");
    printf("3. Perkalian\n");
    printf("4. Pembagian\n");
    printf("5. Exit\n");

    printf("Masukkan Pilihan:");
    scanf("%f", &a);

    if (a > 5){
        printf("Input anda salah, silahkan coba lagi");
        break;
    }
    else if(a == 5){
        printf("Terima kasih, telah menggunakan kalkulator MUHAMMAD RIZKI RAMADHAN");
        break;
    }
    printf("Masukkan nilai pertama:");
    scanf("%f", &b);
    printf("Masukkan nilai kedua:");
    scanf("%f", &c);
    if (a == 1){
        printf("Hasil penjumlahan antara %.2f dengan %.2f adalah %.2f\n", b, c, b + c);
    }
    else if (a == 2){
        printf("Hasil pengurangan antara %.2f dengan %.2f adalah %.2f\n", b, c, b - c);
    }
    else if (a == 3){
        printf("Hasil perkalian antara %.2f dengan %.2f adalah %.2f\n", b, c, b * c);
    }
    else if (a == 4){
        printf("Hasil pembagian antara %.2f dengan %.2f adalah %.2f\n", b, c, b / c);
    }
    }

    return 0;
}

File: PRAK404-2310817310008-MUHAMMADRIZKIRAMADHAN.py
while (1) :
    print("Pilih program")
    print("1. Penjumlahan")
    print("2. Pengurangan")
    print("3. Perkalian")
    print("4. Pembagian")
    print("5. Exit")

    a = int(input("Masukkan pilihan:"))
    if (a > 5):
        print("Input Anda salah, silahkan coba lagi")
        break
    elif (a == 5):
        print("Terima kasih, telah menggunakan kalkulator MUHAMMAD RIZKI RAMADHAN")
        break


    b = int(input("Masukkan nilai pertama:"))
    c = int(input("Masukkan nilai kedua:"))

    if (a == 1) : print(f"Hasil penjumlahan antara {b:.2f} dan {c:.2f} adalah {b + c:.2f}")
    elif (a == 2) : print(f"Hasil pengurangan antara {b:.2f} dan {c:.2f} adalah { b - c:.2f}")
    elif (a == 3) : print(f"Hasil perkalian antara {b:.2f} dan {c:.2f} adalah {b * c:.2f}")
    elif (a == 4) : print(f"Hasil pembagian antara {b:.2f} dan {c:.2f} adalah {b / c:.2f}")